instill-ai/model-backend
# chore: adopt new port

Because

- we have many backends now working together under either docker-compose env or local `make dev` env, to avoid port conflict, we now fix their port numbers. The same setup has been made in https://github.com/instill-ai/vdp/pull/103.

This commit

- adopt the latest checkfield in package `x`
- change the default server port to `8083`
- refine Makefile and docker-compose.yml
- correct test messages
- add semantic PR title check GitHub Actions workflow (remove the old GitHub App)
